<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ColorSelectionWidget</class>
 <widget class="QWidget" name="ColorSelectionWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>398</width>
    <height>52</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item row="0" column="1">
    <widget class="ColorComponentWidget" name="wgtC2" native="true">
     <property name="minimumSize">
      <size>
       <width>128</width>
       <height>14</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>14</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="ColorComponentWidget" name="wgtC1" native="true">
     <property name="minimumSize">
      <size>
       <width>128</width>
       <height>14</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>14</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="0" column="2">
    <widget class="ColorComponentWidget" name="wgtC3" native="true">
     <property name="minimumSize">
      <size>
       <width>128</width>
       <height>14</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>14</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="ColorComponentWidget" name="wgtCAlpha" native="true">
     <property name="minimumSize">
      <size>
       <width>128</width>
       <height>14</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>14</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="2">
    <widget class="QLineEdit" name="wgtHex">
     <property name="inputMask">
      <string>\#HHHhhhhh</string>
     </property>
     <property name="placeholderText">
      <string>hex</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ColorComponentWidget</class>
   <extends>QWidget</extends>
   <header>wgt/colorcomponentwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
